README.md

```diff
@@ -1 +1,205 @@
-http://jgeewax.github.io/phoenix
+# [Phoenix](http://jgeewax.github.io/phoenix)
+
+## Getting Started
+
+Running Phoenix locally is very simple, you'll need `npm`, `bower` and `gulp` 
+installed.
+
+Clone the repository.
+
+```sh
+$ git clone git@github.com:jgeewax/phoenix.git
+```
+
+Install all the project dependencies
+
+```sh
+$ npm install && bower install
+```
+
+Then start the server
+
+```sh
+$ gulp serve
+```
+
+And you're good to go!
+
+## Developing a Plugin
+
+Phoenix is still in a very early stage, so the process for creating plugins is
+still very young and subject to change.
+
+### Create repo for your plugin
+
+**NOTE:** Currently we only support plugins hosted on public GitHub
+repositories.
+
+Your plugin name should be very specific to what it does, since we're using
+Angular 1.x modules for plugins we need to avoid naming collisions. For example,
+if your plugin is named `storage-browser` you should prefix any
+controller/view/services with that name. (`StorageBrowserCtrl`, etc.)
+
+You can see a list of currently available plugins
+[here](https://github.com/jgeewax/phoenix/blob/master/src/plugins.json)
+
+### Create a `package.json` file
+
+Phoenix will parse your `package.json` in order to load your plugin files.
+Your files should be specified in the `files` key.
+
+```javascript
+{
+  "files": [
+    "dist/storage-browser.js",
+    "dist/storage-browser.css"
+  ]
+}
+```
+
+Currently we support, html, css and js. We recommend concatenating your files
+for fewer network requests.
+
+### Create your module
+
+Phoenix uses [System.js](https://github.com/systemjs/systemjs) under the hood to
+import your plugin files, so you'll want to export your angular module.
+
+```javascript
+var mod = angular.module('my-plugin', []);
+
+if (typeof module !== 'undefined' && module.exports) {
```

Comment:
Well, I'm not married to System.js or anything, so if we want to try and cook up something that removes this, that's fine by me :)